### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130818%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#4] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130818%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

讚美主

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130818%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
『因祂愛你的列祖，所以揀選他們的後裔，用祂的同在和大能領你出了埃及，』申四37

『乃因耶和華愛你們，又因要守祂向你們列祖所起的誓，就用大能的手領你們出來，把你們從為奴之家，從埃及王法老的手中救贖出來。』申七8

**Comment by cit-hall-43** [comment]
逾越的羊羔也要與苦菜同喫。苦菜表徵為罪懊悔、悔改的痛苦感覺。用苦菜與逾越的羊羔同喫，意思就是我們需要懊悔、悔改，為罪惡的事經歷痛苦的滋味。逾越節的羊羔一點不可留到次日早晨。這指明我們該享受新鮮的逾越羊羔，也指明我們要完全接受基督，不可局部接受。我們不該讓基督的任何部分剩下，而該完全接受。
